Some mobile optimizations

daniele-athome opened the pull request:
First of all, thanks for this amazing project. I've been using Koel for a while now and I'd like to contribute something - this is a start, I've never played with SASS before and I've never used Vue.js, so I apologize if I've made some mad things :-)
I heavily use it from my phone and I've been frustrated by the footer, so here is a PR with a couple of modifications:

* seek bar thicker
* other-controls bigger (1.2rem)

It **hugely** improves the user experience on a small screen. What do you think?

Tested on Android 4.4 with Firefox and Chrome.

Signed-off-by: Daniele Ricci <daniele.athome@gmail.com>

phanan commented on resources/assets/sass/app.scss:
```diff
@@ -10,8 +10,21 @@ $plyr-control-color: $colorHighlight;
+
 @import "vendor/_plyr.scss";
 
+.plyr {
```
I haven't tested the PR yet, but can't we move this to `index/footer.vue` instead?

daniele-athome replied:
No problem, but did you mean `site-footer/index.vue` maybe?

phanan replied:
Exactly :)

On Sun, Jun 12, 2016 at 6:33 PM, Daniele Ricci <notifications@github.com>
wrote:

> In resources/assets/sass/app.scss
> <https://github.com/phanan/koel/pull/356#discussion_r66719167>:
>
> >  @import "vendor/_plyr.scss";
> >
> > +.plyr {
>
> No problem, but did you mean site-footer/index.vue maybe?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/phanan/koel/pull/356/files/825b5eed351b3ea2bf74747b7e45f3edd8c8ab84#r66719167>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHrt0nP_o4ppoX1wNa0qsoRT5N8CZzRoks5qK-CKgaJpZM4Izlab>
> .
>

daniele-athome commented:
Done! But I had to hard-code the value this time (6*4) because the variable was out of scope somehow. I guess it's not a problem because modifying the seek bar height needs to modify other values as well, it's not just connected to plyr control spacing.

daniele-athome commented:
In the meantime, a picture is worth a thousand words (in this case, two thousands):
![mobile](https://cloud.githubusercontent.com/assets/3763726/15991990/e709f878-30c1-11e6-8a2c-162382d980f2.png)
![desktop](https://cloud.githubusercontent.com/assets/3763726/15991991/e70ad766-30c1-11e6-800a-bba1087cc86c.png)

I'm myself not totally convinced about the seek bar aspect on the desktop version. I mean for mobile it's perfect, but for desktop... I don't know, it seems "too squared"... what do you think?

phanan commented:
Hmm... I'd prefer keeping the desktop version as-is. Also, let me find some time to test the mobile version (both functionality and visual). Thanks!

phanan commented:
I think 24px is a bit too much. 16px would be more balanced.

daniele-athome commented:
> I think 24px is a bit too much. 16px would be more balanced.

Fine by me. I'll rebase my modifications on top of master (including the media query for applying it only to mobile viewport size).

daniele-athome commented:
I've rebased my modifications onto master and also tidied them up a little bit.

* added $footerHeightMobile
* show thicker seek bar (16px) only on mobile viewport

daniele-athome commented:
I forgot about the equalizer which is also using `$footerHeight`. Just pushed again with the equalizer now using `$footerHeightMobile` in the media query block.

phanan commented:
Unfortunately, this is how it looks on iPhone 5 portrait mode:

![image](https://cloud.githubusercontent.com/assets/8056274/16118738/11beda56-340b-11e6-839b-eae2bdcc1bc6.png)

Even worse when playing:

![image](https://cloud.githubusercontent.com/assets/8056274/16118769/2a3602a8-340b-11e6-82e3-bb77af8d1a3b.png)

daniele-athome commented:
I'm sorry but I don't have access to iOS either via emulation or device whatsoever :-( what can we do about that? Is there some online platform that "emulates" iOS Safari that I can test on?

phanan commented:
You can simply use Chrome's Device Toolbar for this.

On Thu, Jun 16, 2016 at 9:45 PM, Daniele Ricci <notifications@github.com>
wrote:

> I'm sorry but I don't have access to iOS either via emulation or device
> whatsoever :-( what can we do about that? Is there some online platform
> that "emulates" iOS Safari that I can test on?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/phanan/koel/pull/356#issuecomment-226489160>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AHrt0iztiUYUNmDtf0-CeCoPdni-C2iaks5qMVNmgaJpZM4Izlab>
> .
>

daniele-athome commented:
Oh, I didn't know it could emulate those with it. I'm on it!

daniele-athome commented:
Fixed and rebased on top of master. I did the best I could, but obviously increasing text size (1.2em) will eventually throw out the repeat button out of the screen on very small viewports (iPhones seem fine though, at least 5 and 6/6+).

phanan commented:
Thanks!

daniele-athome commented:
Thank you for merging! More patches are on their way :-)

phanan commented:
Looking forward to them! Be sure to pull, though – I've made some very
extensive changes over the course of this weekend.

On Sun, Jun 26, 2016 at 7:02 PM, Daniele Ricci <notifications@github.com>
wrote:

> Thank you for merging! More patches are on their way :-)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/phanan/koel/pull/356#issuecomment-228595740>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AHrt0mMCUm1-V-GiBa2hjvfHG-QFmIcrks5qPlxegaJpZM4Izlab>
> .
>
